eng/targets/TargetFrameworks.props

```diff
@@ -13,7 +13,7 @@
   -->
   <PropertyGroup>
     <NetRoslyn>net9.0</NetRoslyn>
-    <NetRoslynAll>net8.0;net9.0</NetRoslynAll>
+    <NetRoslynAll>net8.0;net9.0;net10.0</NetRoslynAll>
```

Comment:
Can we get rid of net9 now, and switch NetRoslyn to net10?  Seems like its mostly for test utilities and things?